docs/TheThingsNetwork.md

```diff
@@ -28,6 +28,16 @@ void reset(bool adr);
 
 - `bool adr`: Enable/disable Adaptive Data Rate.
 
+## Method: `hardReset`
+
+Performs a hardware reset of the RN module. Input parameter is the pin which the reset pin from the module is connected to. This does clear saved state, e.g. provisioned keys.
+
+```c
+void hardReset(uint8_t resetPin);
+```
+
+- `uint8_t resetPin`: The output pin that is connected to the module's reset pin.
+
```

Comment:
We are assuming here that the user already did a pinMode(resetPin, OUTPUT); previously in their code. Let's add to the documentation that the resetPin that is passed to the `hardReset()` function should be a configured output pin.

Reply:
Good point here. Will add this.